### PR TITLE
Adding Clear and NIR accessors

### PR DESCRIPTION
**Pull request by bjones14**

### CODE REVIEW

File: adafruit_as7341.py [hunk 1]
```diff
@@ -375,8 +375,18 @@ def channel_680nm(self):
...
+      
+    @property
+    def channel_clear(self):
+        """The current reading for the clear sensor"""
+        self._configure_f5_f8()
+        return self._channel_4_data
 
-    # TODO: Add clear and NIR accessors
+    @property
+    def channel_clear(self):
```
Comment (caternuson) [inline adafruit_as7341.py]: This one should be named `channel_nir` ?

**Review by caternuson — CHANGES_REQUESTED** [review]

**Comment by caternuson** [comment]
This is for #18?

Were you able to test this code?

File: adafruit_as7341.py (hunk 1 above)
Comment (bjones14) [reply adafruit_as7341.py]: You are right, let me fix that

**Review by bjones14 — COMMENTED** [review]
fixed accessor for channel_nir

**Comment by bjones14** [comment]
> This is for #18?
> 
> Were you able to test this code?

I have not tested this code yet, but I will try to tonight to verify - since I'm just adding accessors for data that was already in the driver, I'm fairly confident it'll work but I will test to verify and report back

**Review by bjones14 — COMMENTED** [review]
updated channel_nir and reviewed changes

**Comment by caternuson** [comment]
It looks like the CI is failing on the formatting check. See here for info on how to run Black:
https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/black

**Review by bjones14 — COMMENTED** [review]
updated file

**Comment by bjones14** [comment]
> It looks like the CI is failing on the formatting check. See here for info on how to run Black:
> https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/black

OK,  the file had CRLF line endings, now has just LF, ran pre-commit on my end and it looks good, pushed new version so should be set to re-run CI tests

**Review by caternuson — APPROVED** [review]

**Pull request merged**